TeamMemberFiles/kaeden.peper.txt

```diff
@@ -0,0 +1,5 @@
+Kaeden Peper
+You can call me Kaeden, no preference. 
+Im in section 294
+I prefer Java, MySQL, and Microsoft
+Im a full time worker and student, but I am excited to take on this course.
```

Comment:
Awesome job Kaeden -
Please make sure to clean up your personal repo to avoid exposing any information to the public you wish to keep private.